sql/wsrep_sst.cc

```diff
@@ -1052,7 +1062,7 @@ static void *sst_donor_thread(void *a) {
   bool locked = false;
 
   const char *out = NULL;
-  int const   out_len = 128;
+  int const   out_len = 1024;
```

Comment:
Is this being used now that logic below directly writes to pipe ?

Reply:
Moved back to command-line args